### PR TITLE
Update Archlinux install instructions

### PR DESCRIPTION
**Pull request by AlexandrePTJ**

### CODE REVIEW

File: docs/installation/arch.rst [hunk 1]
```diff
@@ -1,20 +1,19 @@
 .. _arch-install:
 
-****************************
-Arch Linux: Install from AUR
-****************************
+**************************************
+Arch Linux: Install from ``[community]``
+**************************************
```
Comment (jodal) [inline docs/installation/arch.rst]: I've noticed a couple of Arch users talking about "[community]", but it seems just as usual to just write "community" or "community repository". Can we use one of those variants and avoid this quite special form, at least in the headers?

Also, the length of the `*` lines doesn't match the text length.

File: docs/installation/arch.rst [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 
 If you are running Arch Linux, you can install Mopidy using the
-`mopidy <https://aur.archlinux.org/packages/mopidy/>`_ package found in AUR.
+`mopidy <https://aur.archlinux.org/packages/mopidy/>`_ package found in ``[community]``.
```
Comment (jodal) [inline docs/installation/arch.rst]: Link is outdated. Correct is https://www.archlinux.org/packages/community/any/mopidy/

Same comment about "[community]" again.

File: docs/installation/source.rst [hunk 3]
```diff
@@ -5,7 +5,7 @@ Install from source
 *******************
 
 If you are on Linux, but can't install :ref:`from the APT archive
-<debian-install>` or :ref:`from AUR <arch-install>`, you can install Mopidy
+<debian-install>` or :ref:`with Pacman  <arch-install>`, you can install Mopidy
```
Comment (jodal) [inline docs/installation/source.rst]: What about "the Arch repo"?

File: docs/running.rst [hunk 4]
```diff
@@ -33,7 +33,7 @@ Init scripts
   <https://github.com/mopidy/mopidy/blob/debian/debian/mopidy.init>`_. For
   more details, see the :ref:`debian` section of the docs.
 
-- The ``mopidy`` package in `Arch Linux AUR
+- The ``mopidy`` package in `Arch Linux
   <https://aur.archlinux.org/packages/mopidy>`__ comes with a systemd init
```
Comment (jodal) [inline docs/running.rst]: Link is outdated. Correct is https://www.archlinux.org/packages/community/any/mopidy/

**Comment by jodal** [comment]
Thanks for updating this :-)

File: docs/installation/source.rst [hunk 5]
```diff
@@ -5,7 +5,7 @@ Install from source
 *******************
 
 If you are on Linux, but can't install :ref:`from the APT archive
-<debian-install>` or :ref:`from AUR <arch-install>`, you can install Mopidy
+<debian-install>` or :ref:`with the Arch Linux repository <arch-install>`, you can install Mopidy
```
Comment (trygveaa) [inline docs/installation/source.rst]: s/with/from/?

Comment (AlexandrePTJ) [inline docs/installation/source.rst]: or just remove the 'with'

**Comment by jodal** [comment]
Squashed into bbfa722af345a536741d04ecea3a588acef0acfe.

**Comment by jodal** [comment]
And c4fc33e5eac3253f2e83b50f88194e6900ca04f2.

**Pull request closed without merging**